.env.example

```
@@ -11,6 +11,11 @@ NEXT_PUBLIC_SITE_URL=https://your-store.com
 NEXT_PUBLIC_STORE_NAME=Spree Store
 NEXT_PUBLIC_STORE_DESCRIPTION=A modern e-commerce storefront powered by Spree Commerce and Next.js.
 
+# Demo-only: external links for hero CTAs and footer (remove for production)
+NEXT_PUBLIC_GITHUB_URL=https://github.com/spree/storefront
+NEXT_PUBLIC_QUICKSTART_URL=https://spreecommerce.org/docs/developer/storefront/nextjs/quickstart
+NEXT_PUBLIC_LEARN_MORE_URL=https://spreecommerce.org
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_LEARN_MORE_URL key should go before the NEXT_PUBLIC_QUICKSTART_URL key

(UnorderedKey)